app/assets/javascripts/editor.js

```diff
@@ -174,13 +174,11 @@ class Editor {
     const previewBtn = $("#toggle-preview-button-" + this.commentFormID);
     const formIdPrefix = this.isSingleFormPage ? "#form-body-" : "#comment-form-body-";
     const commentFormBody = $(formIdPrefix + this.commentFormID);
-    // const toolbar = $('.btn-toolbar')[0];
-    const toolbar = commentFormBody.parentElement.children[3]
+    commentFormBody.parent().children().eq(3).toggle();
```

Comment:
This is perfect @KarishmaVanwari. I'm a bit concerned about using the index of a child because it's likely that more elements will be added and then that value will have to change. Since you already have the parent, you could just use the [find()](https://api.jquery.com/find/) method to search for `.btn-toolbar`. What do you think?

Also, can you add a comment like `Toggles toolbar` so that anyone reading the code will know what it's for? You can also move line 177 to any line between lines 179-181 to keep the coding style consistent.

Thank you so much @KarishmaVanwari. You're doing an amazing job!!

Reply:
Thank you so much @TildaDares , you keep motivating me to do better, I'm really glad.

Reply:
And yes, I'll make the changes as you said.

Reply:
@TildaDares I've done the changes. Please have a look.